lib/ads/client_side_ad_manager.js

```diff
@@ -74,6 +77,7 @@ shaka.ads.ClientSideAdManager = class {
   requestAds(imaRequest) {
     goog.asserts.assert(imaRequest.adTagUrl.length,
         'The ad tag needs to be set up before requesting ads.');
+    this.requestAdsStartTime_ = Date.now() / 1000;
```

Comment:
If several ad requests are sent during the playback, only each subsequent load time will replace the previous ones in the stats. Is this what you intended?

Reply:
I had not thought in this case, I will make the time additive for this.

Reply:
I already made the change

Reply:
Does the new version just sum all the request times together?
Lets make it into an array of values or something:
this.loadTimes_ = [];
<...>
  addLoadTime(seconds) {
    this.loadTimeSeconds_.push(seconds);
  }

I'm not sure if it's an actual use case that people care about, so I don't think we need anything more complicated than this at this point.

Reply:
Ok, I can change it. A question, do you want the statistic to return an array or just the sum?

Reply:
Array! Otherwise it'll be the same as now :)
The idea is for people to be able to track how much each request took if they have several.

Thanks!

Reply:
I changed it with the latest commit. Is enough for you?